Assets/HoloToolkit/ARCA/Scripts/Recording/ShowRecordingControls.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using UnityEngine;
+using HoloToolkit.Unity.InputModule;
+
+namespace  ARCA
```

Comment:
namespace does not match the other classes (they are using HoloToolkit.ARCapture)